### PR TITLE
Adding staging task

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
CC @flukeout

File: README.md [hunk 1]
```diff
@@ -32,3 +32,19 @@ This will install all dependencies, build the code, start a server at [http://12
...
+
+#### Production
+
+Any commits to the `master` branch on this repo will trigger an automatic build and deploy to [GitHub Pages](https://mozilla.github.io/womenandweb/).
+
+#### Staging
+
+There is currently no common staging environment for this project. However, you can easily deploy a staged build to your own fork using the following command: `npm run stage REMOTE`. Be sure to substitute `REMOTE` with the name of your personal remote. Also, be sure not to have `npm start` or `npm server` running when you run the staging command.
+
+Once this task completes, your currently checked out branch will be available at:
```
Comment (gvn) [inline README.md:46]: @flukeout – Is this good as additional explanation? Anything else I should add?

Comment (flukeout) [reply README.md:46]: Looks good!

**Comment by gideonthomas** [comment]
This looks good, I'll follow up with an automation process on PRs that uses this.

**Pull request merged**